controller/consul.go

```diff
@@ -52,7 +53,11 @@ func NewConsulMon(addr string) (*ConsulMon, error) {
 
 func (c *ConsulMon) queryServices() ([]*App, error) {
 	var apps []*App
-	addr := c.addr + fmt.Sprintf("%s/%s", nodeURL, c.node)
+	stale := ""
+	if os.Getenv(allowStale) == "true" {
+		stale = "stale"
+	}
+	addr := c.addr + fmt.Sprintf("%s/%s?%s", nodeURL, c.node, stale)
```

Comment:
should this be `?stale=true` instead ?

Reply:
It's just `?stale`.

https://www.consul.io/api/features/consistency.html
https://github.com/hashicorp/consul/issues/1082

Reply:
unfortunately not, the query param is just `stale` which is kind of odd.